todo/script.js

```diff
@@ -0,0 +1,84 @@
+"use stric"
+/* вывод списка дел , второе зачеркиваение 
+статус задачи done: bool;
+id: string;
+
+чтобы вывести список нам нужно вывести элементы массива
+чтобы удалить находим id и изменяем с try на false
+*/
+
+// учим работать с контейнером todos
+const todosNode = document.querySelector('.js-todos');
+const inputNode = document.querySelector('.js-input');
+const btnNode = document.querySelector('.js-btn');
+let todos = [];
+
+function addTodo(text) {
+   const todo = {
+      text,
+      done: false,
+      id: `${Math.random()}`
+   };
+
+   todos.push(todo);
+} // добавление тудушки 
+
+// удаление тудушки 
+
+function deleteTodo(id) {
+   todos.forEach(todo => {
+      if (todo.id === id) {
+         todo.done = true; 
+      }
+   })
+}
+
+function render() {
+   console.log(todos);
+   let html = '';
+
+   todos.forEach(todo => {
+      if (todo.done) {
+         return;
+      };
+      // дописываем удаление и создаем checkbox
+      html += `
+         <div>
+            <input type='checkbox' class='check'>
+            ${todo.text}
+            <button data-id='${todo.id}'>delete</button>
+         </div>
```

Comment:
лучше сделать через li
и data-id сделать для li, а в хендлерах, чтобы получить айди нужно будет li (`event.target.closest('li')`) и у li взять id уже через dataset
```suggestion
         <li data-id='${todo.id}'>
            <input type='checkbox' class='check'>
            ${todo.text}
            <button data-id='${todo.id}'>delete</button>
         </li>
```